medusa/tv/series.py

```diff
@@ -2060,7 +2083,7 @@ def __unicode__(self):
         to_return += u'anime: {0}\n'.format(self.is_anime)
         return to_return
 
-    def to_json(self, detailed=True):
+    def to_json(self, detailed=False, episodes=False):
```

Comment:
My reasoning to have this False by default. Is that I want to have a minimal call to the series object by default. If you need more detailed info you have to opt-in.